bindata/manifests/daemon/daemonset.yaml

```
@@ -70,6 +70,21 @@ spec:
             - name: os-release
               mountPath: /host/etc/os-release
               readOnly: true
+        - name: rdma-cni
+          image: {{.RDMACNIImage}}
+          command:
+            - /bin/sh
+            - -c
+            - cp /usr/bin/rdma /host/opt/cni/bin/
+          securityContext:
+            privileged: true
+          resources:
+            requests:
+              cpu: 10m
+              memory: 10Mi
+          volumeMounts:
+            - name: cnibin
+              mountPath: /host/opt/cni/bin
```

Can we make this block optional? something like:
```suggestion
       {{- if ne .RDMACNIImage "" }}
        - name: rdma-cni
          image: {{.RDMACNIImage}}
          command:
            - /bin/sh
            - -c
            - cp /usr/bin/rdma /host/opt/cni/bin/
          securityContext:
            privileged: true
          resources:
            requests:
              cpu: 10m
              memory: 10Mi
          volumeMounts:
            - name: cnibin
              mountPath: /host/opt/cni/bin\
          {{- end }}
```

This would significantly reduce (or at least postpone) the effort to maintain this in openshift.

---

done